backend/tracim_backend/exceptions.py

```diff
@@ -463,6 +463,10 @@ class TooShortAutocompleteString(TracimException):
     pass
 
 
+class CannotUseBothIncludeAndExcludeWorkspaceUsers(TracimException):
```

Comment:
If this error occured in endpoint , we get a error 500, right ?
I suggest you to either does a marshmallow check like this one https://github.com/tracim/tracim/blob/3fde6f6abaf7401f890a11d05bd680d5395ee717/backend/tracim_backend/views/core_api/schemas.py#L394
or add : 
- specific error code
- catching of this error as 400 in correct endpoint

Reply:
done, using:

```
@hapic.handle_exception(CannotUseBothIncludeAndExcludeWorkspaceUsers, HTTPStatus.BAD_REQUEST)
```

Reply:
nice.